x-aeson/src/X/Data/Aeson.hs

```diff
@@ -10,15 +11,18 @@ module X.Data.Aeson (
   , printPair
   ) where
 
-import           Control.Applicative
-
+import           Control.Applicative ((<*>), pure)
+import           Control.Monad ((>>=), fail)
 import           Data.Aeson as X
 import           Data.Aeson.Types as X
+import           Data.Either (Either(..), either)
+import           Data.Functor ((<$>))
+import           Data.Maybe (Maybe, maybe)
 import           Data.Monoid
 import           Data.Text
 import           Data.Text.Encoding (decodeUtf8, encodeUtf8)
 import           Data.ByteString.Lazy (toStrict)
-
+import           GHC.Base ((.), ($))
```

Comment:
It probably doesn't matter, but I think we should import these from `Data.Function` instead. The `GHC.*` modules are really internal implementation details.